HedgeLib/src/archives/hl_pacx.c

```diff
@@ -1866,14 +1962,24 @@ static HlResult hlINPACxV4DecompressToBlobs(HlBlob* HL_RESTRICT pac,
         /* Setup split PAC entries, decompressing if necessary. */
         for (i = 0; i < splitTable->splitCount; ++i)
         {
-            const HlPACxV4Chunk* chunks = (const HlPACxV4Chunk*)
-                hlOff64Get(&splitEntries[i].chunksOffset);
+            if (isV403)
+            {
+                result = hlPACxV403DecompressBlob(HL_ADD_OFF(header,
+                    splitEntriesV403[i].offset), splitEntriesV403[i].compressedSize, splitEntriesV403[i].uncompressedSize,
+                    &((*pacs)[*pacCount]));
+            }
+            else 
+            {
```

Comment:
Same here

```c
HlPACxV401SplitEntry* splitEntriesV401 = (HlPACxV401SplitEntry*)splitEntries;
```

All the references to splitEntries in this scope should also be changed to splitEntriesV401 ofc